edsdme/components/NewsCard.js

```diff
@@ -24,6 +26,26 @@ class NewsCard extends LitElement {
     return newUrl;
   }
 
+  checkBackgroundImage(element) {
+    const style = window.getComputedStyle(element);
```

Comment:
@richardhand-fortedigital Maybe, instead of slicing here, we can directly use
`const url = this.data.styles?.backgroundImage;`
since it is already passed in the HTML `...style="background-image: url('${this.data.styles?.backgroundImage}')"...`

Reply:
great suggestion! Thanks @draganatrajkovic 

updated in https://github.com/adobecom/dme-partners/pull/28/commits/e75ab5412c3199be143adb6de92a1815ba7f0f2d